RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitHBHE_cfi.py

```diff
@@ -9,9 +9,8 @@
 
 particleFlowRecHitHBHE = cms.EDProducer("PFRecHitProducer",
     navigator = cms.PSet(
-            name = cms.string("PFRecHitHCALNavigator"),
-            sigmaCut = cms.double(4.0),
-            timeResolutionCalc = _timeResolutionHCAL
```

Comment:
If this PSet is not needed any more, then probably also the corresponding import can be removed:
```
from RecoParticleFlow.PFClusterProducer.particleFlowCaloResolution_cfi import _timeResolutionHCAL
```

Moreover, there are other such imports that could be also dropped, see https://cmssdt.cern.ch/lxr/search?%21v=CMSSW_11_1_X_2020-03-02-2300&_filestring=&_string=particleFlowCaloResolution , as well as the corresponding unused config parameters.
That could be deferred to another cleaning PR, though